docs/maintainer/knowledge_base.md

```diff
@@ -1162,9 +1162,16 @@ corresponding compilers in `requirements/build` as normal.
 
 ### OpenMP
 
-You can enable OpenMP on macOS by adding the `llvm-openmp` package to the `build` section of the `meta.yaml`.
-For Linux OpenMP support is on by default, however it's better to explicitly depend on the libgomp package which is the OpenMP
-implementation from the GNU project.
+The default OpenMP runtime used depends on the compiler and platform used. GCC (on Linux and the Windows MinGW
+toolchain) uses the `libgomp` runtime. Clang (on all platforms) and GNU Fortran on macOS default to the `llvm-openmp`
+runtime. MSVC may use its own runtime or `llvm-openmp`. However, individual projects may force building against
```

Comment:
Is this worth reflecting in the `meta.yaml` example below?

Reply:
How do individual projects force building against a different runtime then?

Reply:
Usually some complex CMake logic (I'm thinking of PyTorch).